src/main/java/uk/gov/justice/hmpps/casenotes/dto/NomisCaseNote.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.hmpps.casenotes.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@ApiModel(description = "Case Note")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@Data
+public class NomisCaseNote {
+
+    @ApiModelProperty(required = true, value = "Case Note Id (unique)", position = 1, example = "12311312")
+    @NotNull
+    private Long caseNoteId;
+
+    @ApiModelProperty(required = true, value = "Offender Unique Identifier", position = 2, example = "A1234AA")
+    @NotNull
+    private String offenderIdentifier;
+
+    @ApiModelProperty(required = true, value = "Case Note Type", position = 3, example = "KA")
+    @NotBlank
+    private String type;
+
+    @ApiModelProperty(required = true, value = "Case Note Type Description", position = 4, example = "Key Worker")
+    @NotBlank
+    private String typeDescription;
+
+    @ApiModelProperty(required = true, value = "Case Note Sub Type", position = 5, example = "KS")
+    @NotBlank
+    private String subType;
+
+    @ApiModelProperty(required = true, value = "Case Note Sub Type Description", position = 6, example = "Key Worker Session")
+    @NotBlank
+    private String subTypeDescription;
+
+    @ApiModelProperty(required = true, value = "Source Type", position = 6, example = "INST")
```

Comment:
two position `6` values

Reply:
although I can't see this being returned by a controller, so therefore validation and model properties can be removed